examples/chip-tool/commands/tests/TestCommand.h

```diff
@@ -40,17 +40,12 @@ class TestCommand : public CHIPCommand
     {
         AddArgument("node-id", 0, UINT64_MAX, &mNodeId);
         AddArgument("delayInMs", 0, UINT64_MAX, &mDelayInMs);
-        AddArgument("timeout", 0, UINT16_MAX, &mTimeout);
```

Comment:
did the timeout configurable feature has been merged ?

Reply:
It has been merged already, yes. But while doing this patch I have noticed a typo. A ':' instead of a '?'...